CHANGELOG.rst

```diff
@@ -6,6 +6,9 @@ I2S library change log
 
   * ADDED: Support for XCommon CMake build system
   * RESOLVED: Added missing shutdown feature to i2s_frame_slave
+  * FIXED: Allow input and output ports in the 4-bit port implementation to be
+    nullable
+  * FIXED: Behaviour of the restart_check() callback function
```

Comment:
Worth noting this is just in examples because that's where the callbacks are defined, the behaviour of the component itself has not changed